### PR TITLE
Dev server negotiation

### PR DESCRIPTION
**Pull request by Cobus-Redelinghuys**
Please note on the User.kt file (User model file), the agreementB property. 
I have a question about this for our clients on how to do this more elegantly. 
It works as is currently.
I updated the postman team thing to include all new changes and additions. 
I tested with both unit tests and postman tests and all works.

### CODE REVIEW

**Review by Kaleb-Bruwer — APPROVED** [review]
Looks good to me

File: Smart-Contract-Verifier-Server/docker/docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,9 @@
 services:
   postgres:
     image: postgres:latest
+    restart: always
     volumes:
-      - ./database:/var/lib/postgresql/data
```
Comment (JasonMaritz) [inline Smart-Contract-Verifier-Server/docker/docker-compose.yml:6]: This was a change by JasonMaritz that is necesarry

File: Smart-Contract-Verifier-Server/docker/scripts/init.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DROP DATABASE IF EXISTS verifierserver;
```
Comment (JasonMaritz) [inline Smart-Contract-Verifier-Server/docker/scripts/init.sql:1]: Again a change by JasonMaritz used to update the database

**Review by JasonMaritz — APPROVED** [review]
Looks good

**Pull request merged**